client/src/components/ActiveTask/MyActive/ActiveModal.jsx

```diff
@@ -15,17 +17,19 @@ justify-content: center;
 
 const ActiveModal = () => {
   const [show, setShow] = useState(false);
-  const [review, setReview] = useState('');
+  const [newReview, setNewReview] = useState('');
```

Comment:
style guide recommends spaces inside square brackets

Reply:
linter doesnt like it with spaces